runtime/server/generate_metrics_view.go

```diff
@@ -195,11 +197,15 @@ func (s *Server) generateMetricsViewYAMLWithAI(ctx context.Context, instanceID,
 			doc.Connector = connector
 		}
 		doc.Table = tblName
+		doc.Database = tbl.Database
+		doc.Schema = tbl.DatabaseSchema
```

Comment:
We really shouldn't do this for cases where people are using the connector's default database/schema. It bloats the metrics YAML, which is already our most complex YAML artifact.

An option to easily check it might be to include two booleans on `drivers.Table` indicating whether the database and schema respectively is the default one.